teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    @specialization = specialization
+    super(age, name, parent_permission: parent_permission)
+  end
```

Comment:
[OPTIOANL]
- [ ] As said early, to be able to add the variable `parent_permission` different than `true`
```suggestion
  def initialize(specialization, age, name = 'unknown', parent_permission = true)
    @specialization = specialization
    super(age, name, parent_permission)
  end
```